lucille-examples/lucille-s3-ingest-example/scripts/run_ingest.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+# run this script from top level lucille-simple-csv-solr-example directory via ./scripts/run_ingest.sh
```

Comment:
comment references lucille-simple-csv-solr-example but this is lucille-s3-ingest-example

Reply:
updated